contracts/v0.2/src/SubmissionProxy.sol

```diff
@@ -92,8 +105,28 @@ contract SubmissionProxy is Ownable {
             revert InvalidOracle();
         }
 
-        whitelist[_oracle] = block.timestamp + expirationPeriod;
-        emit OracleAdded(_oracle);
+	uint256 expirationTime_ = block.timestamp + expirationPeriod;
```

Comment:
Since Solidity 0.8, there is internal detection for underflow/overflow. No need to add SafeMath. What happened to you today? Did you have fun weekend? :D